docs/shared-content/structured-configuration-variables.include.md

```diff
@@ -0,0 +1,224 @@
+:::info
+This Configuration Feature used to be called JSON Configuration Variarbles but has been re-named with the added support for YAML
```

Comment:
Passing on [previous advice from Lee](https://octopusdeploy.slack.com/archives/C44HGJF8D/p1594013919086700):

- [ ] Consider be more specific about the version when this changed, e.g.:

> 
> :::warning
> In Octopus 2020.3 and earlier this feature was called ....
> :::

- [ ] We’ll also need to update octopurls as I believe there are a few links in the product to the docs

Reply:
Good point about specificity :) 

I might be stating the obvious here, but just in case, we don't need to update Octopurl with links to .include.md files only the .md files that include those .include.md file if the .md filename/location has changed :)